decidim-generators/lib/decidim/generators/app_templates/initializer.rb

```diff
@@ -347,8 +349,76 @@
   config.allow_open_redirects = Rails.application.secrets.decidim[:allow_open_redirects] if Rails.application.secrets.decidim[:allow_open_redirects].present?
 end
 
+if Decidim.module_installed? :api
+  Decidim::Api.schema_max_per_page = Rails.application.secrets.dig(:decidim, :api, :schema_max_per_page).presence || 50
```

Comment:
I suggest that for the module configurations we would use similar configuration blocks as we are using for the Decidim core.

E.g. for the API engine:

```ruby
Decidim::Api.configure do |config|
  config.schema_max_per_page = Rails.application.secrets.dig(:decidim, :api, :schema_max_per_page).presence || 50
  # etc.
end
```

Reply:
Right, much cleaner. Do you think each module should have and independent initializer file in addition?

Reply:
Some 3rd party modules have their own initializer files but I can also see the core gems being in the same initializer.

So for me it's also fine as-is, if we just create the configuration blocks.